src/frontend/src/components/Banner/Banner.css

```diff
@@ -12,11 +12,3 @@
   height: 100vh;
   opacity: 0.4;
 }
-
```

Comment:
In this or another follow-up PR, we should drop the rest of this CSS, and move it into our component JS.

Reply:
All for removing the last remaining CSS here, but let's land #802 first since I'd have to update those classes first. Fair?

Reply:
Sold.